tests/Unit/CheckoutTest.php

```diff
@@ -583,4 +585,13 @@ public static function successDeleteStoredPaymentMethodsProvider()
             array('tests/Resources/Checkout/deleteStoredPaymentMethods-success.json', 200),
         );
     }
+
+    public function testDoublePathParamSubstitution()
+    {
+        $client = $this->createMockClientUrl('tests/Resources/ModelBasedCheckout/sessions-success.json');
+        $resource = new Checkout($client);
+        $resource->captures(array('paymentPspReference' => 'pspRef1'));
```

Comment:
I don't think we need to capture the first call in a var. this is a little bit cleaner.
```suggestion
        captures(array('paymentPspReference' => 'pspRef1'));
```